ratis-test/src/test/java/org/apache/ratis/grpc/TestSnapshotManagementWithGrpc.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.ratis.grpc;
+
+import org.apache.ratis.server.impl.MiniRaftCluster;
+import org.apache.ratis.statemachine.SnapshotManagementTest;
+
+public class TestSnapshotManagementWithGrpc extends SnapshotManagementTest {
```

Comment:
@codings-dan , We may implements MiniRaftClusterWithGrpc.FactoryGet in TestSnapshotManagementWithGrpc, i.e.
```
-import org.apache.ratis.server.impl.MiniRaftCluster;
 import org.apache.ratis.statemachine.SnapshotManagementTest;
 
-public class TestSnapshotManagementWithGrpc extends SnapshotManagementTest {
-  @Override
-  public MiniRaftCluster.Factory<? extends MiniRaftCluster> getFactory() {
-    return MiniRaftClusterWithGrpc.FACTORY;
-  }
+public class TestSnapshotManagementWithGrpc
+    extends SnapshotManagementTest<MiniRaftClusterWithGrpc>
+    implements MiniRaftClusterWithGrpc.FactoryGet {
 }
```
Since it is very minor, I just merged this pull request.  Could you change it in the next pull request?

Reply:
Thanks for reviewing code, will change code in the next pull request, thx!